api/services/AccessControlService/index.ts

```diff
@@ -1,6 +1,6 @@
 import acl from './acl';
 import initialize from './initialize';
-export * from './getEventRoles';
+export * from './EventAccessControl/getEventRoles';
```

Comment:
You want to export everything in `EventAccessControl` and `NewsAccessControl`.

I recommend you to create an `index.ts` files in each of the folders.

Reply:
For your reference, you can look at `@Controllers/EventController/index.ts`.